mm2src/mm2_main/src/lp_ordermatch/orderbook_rpc.rs

```diff
@@ -351,7 +366,10 @@ pub async fn orderbook_rpc_v2(
                         continue;
                     },
                 };
-                let is_mine = my_pubsecp == bid.pubkey;
+                let is_mine = match &my_pubsecp {
```

Comment:
It can be one line
```rust
let is_mine = my_pubsecp.as_ref().map(|my| my == &bid.pubkey).unwrap_or(false);
```

Please change in other similar places too. Maybe it's even worth making it a function to avoid code duplication.